akka-stream/src/main/scala/akka/stream/javadsl/package.scala

```diff
@@ -6,6 +6,8 @@ package akka.stream
 package object javadsl {
   def combinerToScala[M1, M2, M](f: akka.japi.function.Function2[M1, M2, M]): (M1, M2) ⇒ M =
     f match {
+      case x if x eq Keep.left   ⇒ scaladsl.Keep.left.asInstanceOf[(M1, M2) ⇒ M]
```

Comment:
Interesting, I never knew about that one.
Seems though that it only is about "stable type", and not generally speaking "assume this is stable": http://www.scala-lang.org/files/archive/spec/2.11/11-annotations.html#scala-compiler-annotations

(wow so nice we actually can link to spec like that nowadays (regardless of its state, I know I know))